types/v1/types.go

```diff
@@ -1304,3 +1305,83 @@ type NFSExportModify struct {
 	AddReadWriteRootHosts    []string `json:"add_read_write_root_hosts,omitempty"`
 	RemoveReadWriteRootHosts []string `json:"remove_read_write_root_hosts,omitempty"`
 }
+
+// UploadPackageParam defines struct for Upload Package
+type UploadPackageParam struct {
+	FilePath string `json:"file_path"`
+}
+
+// PackageDetails defines struct for Package Details Response
+type PackageDetails struct {
+	Filename        string `json:"filename"`
+	OperatingSystem string `json:"operatingSystem"`
+	LinuxFlavour    string `json:"linuxFlavour"`
+	Version         string `json:"version"`
+	SioPatchNumber  int    `json:"sioPatchNumber"`
+	Label           string `json:"label"`
+	Type            string `json:"type"`
+	Size            int    `json:"size"`
+	Latest          bool   `json:"latest"`
+}
+
+// GatewayResponse defines struct for Gateway API Response
+type GatewayResponse struct {
+	Message    string `json:"message,omitempty"`
+	Data       string `json:"data,omitempty"`
+	StatusCode int    `json:"httpStatusCode,omitempty"`
+	ErrorCode  int    `json:"errorCode,omitempty"`
+}
+
+// MDMTopologyParam defines struct for Validate MDM Topology
+type MDMTopologyParam struct {
+	MdmIps                []string `json:"mdmIps"`
+	MdmUser               string   `json:"mdmUser"`
+	MdmPassword           string   `json:"mdmPassword"`
+	SecurityConfiguration struct {
```

Comment:
Instead of  creating the struct inline can you create a  new separate struct and place the name of the struct here.

Reply:
@VamsiSiddu-7 I created inline because no use separate use of that inline struct.

Reply:
But i am thinking it is good for readability if we have a separate struct.